src/components/common/GProfilerModal.tsx

```diff
@@ -0,0 +1,465 @@
+                                                    topNumFeaturesValue
+                                                );
+
+                                                return (
+                                                    <Table.Row>
+                                                        <Table.Cell collapsing>
+                                                            <Checkbox
+                                                                onChange={
+                                                                    handleToggleTopNumFeatures
+                                                                }
+                                                            />
+                                                        </Table.Cell>
+                                                        <Table.Cell>
+                                                            {isSelected ? (
+                                                                <b>
+                                                                    {`Top ${topNumFeaturesValue}`}
+                                                                </b>
+                                                            ) : (
+                                                                <span
+                                                                    style={{
+                                                                        textDecorationLine:
+                                                                            'line-through',
+                                                                        textDecorationStyle:
+                                                                            'solid',
+                                                                    }}>{`Top ${topNumFeaturesValue}`}</span>
+                                                            )}
+                                                        </Table.Cell>
+                                                    </Table.Row>
+                                                );
+                                            }
+                                        )}
+                                    </Table.Body>
+                                </Table>
+                                <Form.Group widths='equal'>
+                                    <Form.Field
+                                        control={Select}
+                                        label='Sort Features By'
+                                        options={this.availableSortBy}
+                                        placeholder='Sort By'
+                                        onChange={this.handleSelectSortBy}
+                                        value={selectedSortBy}
+                                    />
+                                    <Form.Field
+                                        control={Select}
```

Comment:
This would be very nice as an `Dropdown` instead of a `Select`, this would let people type to filter the list.